### ISSUE TITLE
Problem with tiny models

### ISSUE THREAD

**Issue by WajWaraich** [AlexeyAB/darknet#14]
Hi guys,

The larger models work perfectly well and produce good results. However, when I run it with the tiny models it produces no output, I use the -c flag and it just shows the raw image with no detections. I have tried reducing the -thresh to a very low number but it still produces no output, when -thresh is set to zero it only shows "person", so I am guessing the model has only been trained using people?

Kind regards,
Waj

**Comment by AlexeyAB** [comment]
@WajWaraich 

Can you mistakenly mixed: `.data`, `.cfg` and `.weights` files of COCO and VOC?

This works well for me: 

* VOC: `darknet.exe detector test data/voc.data tiny-yolo-voc.cfg tiny-yolo-voc.weights -thresh 0.3`
* COCO: `darknet.exe detector test data/coco.data tiny-yolo.cfg tiny-yolo.weights -thresh 0.3`

Try on this image: https://hsto.org/files/f1d/173/ca2/f1d173ca2ff143058a700ea609178773.jpg

* VOC: 
 ![predictions](https://cloud.githubusercontent.com/assets/4096485/22546954/5400eba2-e950-11e6-8acc-6031e74a1ef8.jpg)

*COCO: 
 ![predictions](https://cloud.githubusercontent.com/assets/4096485/22547135/4e2ed74c-e951-11e6-953c-a825b0a16204.jpg)

**Comment by WajWaraich** [comment]
@AlexeyAB 

Thanks for your prompt reply, I have been using the following command:
`darknet.exe detector demo data/coco.data tiny-yolo.cfg tiny-yolo.weights`

I tried using `test` instead:
`darknet.exe detector test data/coco.data tiny-yolo.cfg tiny-yolo.weights` but got no output. I am not sure what I am doing wrong.

![predictions2](https://cloud.githubusercontent.com/assets/5938514/22547909/e7fe8342-e93b-11e6-9c96-ff6b4eafb9e0.PNG)

**Comment by AlexeyAB** [comment]
@WajWaraich 

1. Did you get the same result on VOC (cfg & weights)?
2. Try to download COCO (cfg & weights) from:
* https://github.com/pjreddie/darknet/blob/master/cfg/tiny-yolo.cfg
* http://pjreddie.com/media/files/tiny-yolo.weights

3. And look into coco.data, there should be:
```
classes= 80
train  = data/coco/trainvalno5k.txt
valid  = data/coco_testdev
#valid = data/coco_val_5k.list
names = data/coco.names
backup = backup/
eval=coco
```

**Comment by WajWaraich** [comment]
@AlexeyAB 
1. Yep, the results were the same for both VOC and COCO.
2. Downloaded the weights and cfg and replaced the old ones with those - didn't help, they were already correct.
3. `coco.data` contains exactly what you said.

This is quite strange, it works fine with the large weights and cfg but doesn't work for the tiny ones.

**Comment by AlexeyAB** [comment]
@WajWaraich Do you use MSVS2015, CUDA 8.0, OpenCV 2.4.9, and compile it with Release & x64?

And do you use cuDNN and what version v5.1?

**Comment by WajWaraich** [comment]
@AlexeyAB 

Yep I use MSVS2015, CUDA 8.0.44, OpenCV 2.4.9 and compile with cuDNN v5.1 following your guide.

**Comment by WajWaraich** [comment]
@AlexeyAB 

I will try recompiling from scratch again and see whether that makes a difference, will update you on my progress.

**Comment by WajWaraich** [comment]
Alright just recompiled it and added the missing .dll files inside the release build, still the same issue - works perfectly with the large models but not with the tiny ones.

![image](https://cloud.githubusercontent.com/assets/5938514/22553165/a24007d2-e953-11e6-9391-a87ae8c8ca76.png)

**Comment by VitaliCzymmek** [comment]
Maybe you should disable CUDNN and try again. Hope this works for your problem?
